examples/pytorch/hgp_sl/networks.py

```diff
@@ -0,0 +1,137 @@
+import torch
+from dgl.nn import AvgPooling, MaxPooling
+import torch.nn.functional as F
+import torch.nn
+from layers import HGPSLPool, WeightedGraphConv, ConvPoolReadout
+
+
+class SimpleModel(torch.nn.Module):
```

Comment:
1. Is `SimpleModel` a special case of `Model`? If so, we may define this class by `Model`.
2. Better rename the classes to something more meaningful, e.g. `HGPSLModel`.

Reply:
The `SimpleModel` is a 3-layer (fixed, cannot change using input parameters) model which is the same as the author's implementation. We use the `Model` class which can set the number of layers from outside. I will remove the redundant `SimpleModel`